cscs-checks/system/basic/file_system_check.py

```diff
@@ -9,117 +9,113 @@
 import re
 
 import reframe as rfm
-import reframe.utility.sanity as sn
 import reframe.utility.osext as osext
+import reframe.utility.sanity as sn
+
+
+class FileSystemGlobal(rfm.RegressionMixin):
+    '''Handy class to store common test settings.
+    '''
+    scratch = parameter(variable(list, value=['SCRATCH']))
```

Comment:
Do you mean the following?

```suggestion
    scratch = parameter(['SCRATCH'])
```

Reply:
Done